src/witan/models/load_data.clj

```diff
@@ -80,6 +80,21 @@
   {:column-names (apply-col-names-schema PopulationSchema csv-data)
    :columns (vec (apply-row-schema PopulationSchema csv-data))})
 
+(defmethod apply-rec-coercion :births
```

Comment:
why not load up the data into a dataset and then just check the dataset w/the schema?